portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,30 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<String> fruitFacts;
+
+  @Override
+  public void init() {
+    fruitFacts = new ArrayList<>();
+    fruitFacts.add("A strawberry is not an actual berry");
+    fruitFacts.add("Kiwis contain more Vitamin C than oranges");
+    fruitFacts.add("The pineapple is actually a berry");    
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Kiwis contain more Vitamin C than oranges :o");
+    // Convert the fruitFacts ArrayList to JSON
+    String json = convertToJsonUsingGson(fruitFacts);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /** Convert an ArrayList instance into a JSON string using the Gson library. */
```

Comment:
I love this helper method! It helps separate everything really nicely! The javadoc is unnecessary though since the method name is self-explanatory (great job btw)

Reply:
Yay! I actually got it from the examples provided which have a couple ways to convert to JSON. I liked this one since it's so simple and makes the most sense to me haha but I will take the compliment XD